project-songfy/src/features/Dashboard/index.jsx

```diff
@@ -25,15 +25,22 @@ const DashboardFeatures = () => {
     getMe();
   }, []);
   return (
-    <div className="bg-black h-screen pt-2 pr-2">
-      <div className="bg-[#1E1E1E] w-full h-[500px] rounded-xl pt-2">
-        <div className="flex gap-2 flex-wrap justify-center m-2">
-          {data?.categories?.items?.slice(0, 18).map((item, index) => (
+    <div className="bg-black w-full pt-2 pr-2">
+      <div className="bg-[#1E1E1E] w-full rounded-xl p-2">
+        <div className=" gap-2 m-2 ">
+          {data?.categories?.items?.slice(0, 4).map((item, index) => (
             <div
               key={index}
```

Comment:
use id on mapping data, see response on console, and use the id spesific item
ex : item?.id